src/Java.Interop.Tools.Cecil/Java.Interop.Tools.Cecil/TypeDefinitionRocks.cs

```diff
@@ -81,6 +81,12 @@ public static string GetAssemblyQualifiedName (this TypeReference type)
 					(def ?? type).Module.Assembly.Name.FullName);
 		}
 
+		public static string GetAssemblyName (this TypeReference type)
```

Comment:
As per previous discussion, this should be named `GetPartialAssemblyName()`, to "mirror" `GetPartialAssemblyQualifiedName()` -- unless we can find a better/more conventional name -- and we should refactor ~all instances of `.Module.Assembly.Name.Name` within this repo to use `GetPartialAssemblyName()`.